sdk/storage/storage-blob/samples/browserSamples/largeFileUploads.html

```diff
@@ -1,109 +0,0 @@
-<!--
-  Copyright (c) Microsoft Corporation. All rights reserved.
-  Licensed under the MIT Licence.
-
-  Here we demonstrate how to inject new SAS for an ongoing uploading. 
-  SAS might get expired before a large upload finishes, for this scenario, we want to request a new SAS token during the upload instead of starting a new upload.
-  
-  In this sample, we give a SAS injection sample for browsers like Chrome which supports await/async.
-  Sample code imports Azure Storage Blob SDK with JavaScript bundle.
-  Before executing the sample: 
-  - Make sure storage account has been set up CORS setting properly
-  - Implement method getNewSasForBlob 
-  - Update url in upload() method
```

Comment:
Can you copy this setup too to the corresponding `.js` file?

Reply:
Yep, will do this now.